packages/kotti-ui/source/kotti-style/index.scss

```diff
@@ -12,6 +12,7 @@
 // Utility classes
 @import 'utilities';
 @import 'media';
+@import 'mixins';
```

Comment:
Not 100% sure, but this looks like a no-op.

Reply:
@FlorianWendelborn can this be resolved?

Reply:
Still seems like a noop

Reply:
can you either elaborate it or test it?